server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/cucumber/CassandraUploadCucumberTest.java

```diff
@@ -27,7 +29,8 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(features = {"classpath:cucumber/UploadEndpoint.feature"},
                 glue = {"org.apache.james.jmap.methods.integration", "org.apache.james.jmap.cassandra.cucumber"},
-                tags = {"not @Ignore"},
+                tags = {"not @Ignore", "@BasicFeature"},
                 strict = true)
+@Category(EnableCucumber.class)
```

Comment:
do you added this category on every cucumber class? Is it needed?

Reply:
yes It is needed, Only categorized classes/methods will be executed by maven surefire. Otherwise, maven surefire will ignore cucumber test classes.